banking-app/test/scripts/Install_ccf.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -euo pipefail
+
+echo "Install CCF" 
+
+echo "Clone CCF Repo to be used in the Prerequisites Installation"
+cd ~/repos
```

Comment:
This can fail if we don't have `repos` folder on our home.

Reply:
Done